app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainViewModel.kt

```diff
@@ -11,11 +11,13 @@ class MainViewModel : ViewModel() {
     }
 
     fun getAllGroups() : ArrayList<Group> {
-        var groups : ArrayList<Group> = ArrayList()
+        val groups : ArrayList<Group> = ArrayList()
         //placeholder, should be replaced by database call/live data
-        groups.add(Group(name = "Group1", desc = "This data is", active = true, same_schedule = false, group_id = 1))
-        groups.add(Group(name = "Group2", desc = "set in the", active = false, same_schedule = false, group_id = 1))
-        groups.add(Group(name = "Group3", desc = "main view model", active = true, same_schedule = false, group_id = 2))
+        with(groups) {
```

Comment:
Implemented with() so there is no need to use same dot operators multiple times

Reply:
Since this is test data, consider moving it to another class entirely.